src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```
@@ -222,6 +225,13 @@ export class CourseExerciseDetailsComponent extends AbstractScienceComponent imp
     handleNewExercise(newExerciseDetails: ExerciseDetailsType) {
         this.exercise = newExerciseDetails.exercise;
 
+        // TODO TW: This "feature" is only temporary for a paper.
+        if (this.exercise.problemStatement?.includes('ICER 2025 Paper')) {
+            this.accountService.identity().then((user) => {
+                this.isChatGptWrapper = user && user.id ? user.id % 3 == 0 : false;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)